File: fotocopiadora.java
import java,util.Scanner;
////////////////////////////////
public class fotocopiadora {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
////////////////////////////////
        System.out.print("\033[H\033[2J")
        System.out.flush()
////////////////////////////////
        System.out.print("ingresa la cantidad de hojas por favor");
        int hojas = scanner.nextInt();
//////////////////////////////////
        int precioPorHoja = 20
        double total = precioPorHoja * hojas
//////////////////////////////////
        if (hojas > 30) {
            total*= 0.90;
        } 
///////////////////////////////////
        System.out.print("presiona pra continuar")
        scanner.nextLine();
        scanner.nextLine();
////////////////////////////////////
        System.out.print("\033[H\033[2J")
        System.out.flush();
/////////////////////////////////////
        System.out.println("*******Boleta******
        cantidad de hojas %d
        valor por hoja: $20
        TOTAL $%.2f
        ".formatted(hojas, total));
///////////////////////////////////////
        scanner.close();                                                               
    }
}
/////////////////////////////////////////

File: fotocopiadora.kt
import java.util.Scanner
////////////////////////////
fun main() {
    val scanner = Scanner(System.`in`)
/////////////////////////////
    print("\u001b[H\u001b[2J")
    System.out.flush()
/////////////////////////////
    print("ingresa la cantidad de hojas por favor")
    val hojas = scanner.nextInt()
//////////////////////////////
    val precioPorHoja = 20
    var total = precioPorHoja * hojas
///////////////////////////////
    if (hojas > 30) {
        total *= 0.90
    }
///////////////////////////////
    print("\n presiona para continuar")
    scanner.nextLine()
    scanner.nextLine()
////////////////////////////////
    print("\u001b[H\u001b[2J")
    System.out.flush()
////////////////////////////////
    println(""" 
    *********boleta********
    cantidad de hojas: $hojas
    valor por hoja: $20
    total: $${"%.2f".format(total)}




    """.trimIndent())
//////////////////////////////////
    scanner.close()                                
}
//////////////////////////////////

